escodegen.js

```diff
@@ -2434,7 +2434,15 @@
 
         ModuleSpecifier: function (expr, precedence, flags) {
             return this.Literal(expr, precedence, flags);
-        }
+        },
+
+        ImportExpression: function(expr, precedence, flag) {
+            return [
+                'import(',
+                this.generateExpression(expr.source),
```

Comment:
```suggestion
                this.generateExpression(expr.source, Precedence.Assignment, E_TTT),
```

so that things like `import(('a', 'b'))` get codegened properly. Please also add a test for that.

Reply:
Makes sense! Fixed in e8f0084.